tests/excel_bugs/README.md

```diff
@@ -0,0 +1,14 @@
+Test excel bugs
+===============
+
```

Comment:
"Excel bugs" is a bit vague about what the test actually does: it looks like it tests for loading excel files with missing values. Can you rename to be descriptive enough that the same tests could be meaningfully applied to other projects?

Reply:
Sure, thank you for your comment.